ci/unit/test_post_always.sh

```diff
@@ -50,3 +50,6 @@ rsync -v -dpt -z -e "ssh $SSH_KEY_ARGS" jenkins@"$NODE":/tmp/ \
 # Note that we are taking advantage of the NFS mount here and if that
 # should ever go away, we need to pull run_test.sh/ from $NODE
 python utils/fix_cmocka_xml.py
+
+cp "${DAOS_BASE}/test_results/*.xml $test_log_dir
```

Comment:
(lint) The mentioned syntax error was in this simple command. [SC1009]
(lint) Couldn't parse this double quoted string. Fix to allow more checks. [SC1073]